premerge/cluster-management.md

```diff
@@ -0,0 +1,99 @@
+# Cluster configuration
+
+The cluster is managed using Terraform. The main configuration is
+[main.tf](main.tf).
+
+---
+NOTE: As of today, only Googlers can administrate the cluster.
```

Comment:
Maybe we should only have this notice in one place?

Reply:
removed it from anywhere, except from the cluster configuration page.